tests/app/autoload.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
For register annotation. See http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/index.html.